types/experiments.ts

```diff
@@ -151,8 +151,8 @@ interface FeatureConfig {
   /** The identifier for the feature flag */
   featureId: string;
 
-  /** Optional extra params for the feature (this should be validated against a schema) */
-  value: { [key: string]: unknown };
+  /** Optional extra data for the feature (this should be validated against a schema) */
+  value: { [key: string]: unknown } | Array<unknown>;
```

Comment:
Hmmm… will this have any backwards compatibility issue? Where an older version of Firefox even if not targeted by the version check might need to process the whole object and could get confused by the array here.

This might just work fine, but I wasn't expecting messaging system value usage to require nimbus changes.

Reply:
I don't think so, it only seems to navigate the object if it has an `id` or `trigger`. But I discussed with Barret and we're thinking it may be better to just use an object with a `messages` property, and avoid changing nimbus at all. If that sounds OK, I will close this and update the patch for m-c